packages/halo-theme/src/palettes/core.less

```diff
@@ -2,5 +2,6 @@
 
 @color-refinitiv-blue  : #001EFF;
 @color-blue-ribbon     : #334BFF;
+@color-persian-blue    : #1429BD;
```

Comment:
Are you sure it is `core` colour? I cannot see it here: https://www.figma.com/file/ZNGihMJU2eu1femBxAcBRp/HALO---DARK----UI-COMPONENT-LIBRARY?node-id=18152%3A21663.

Reply:
Thanks for your suggestion aleks, after i ask @dtanp-rft , i will move it to secondary.less

Reply:
updated